src/components/TaskList/TaskList.jsx

```diff
@@ -210,13 +210,29 @@ class TaskList extends React.Component {
           </div>
         </nav>
         <div className="task-list-container">
-          <ol className="task-list">
-            {this.props.tasks.map((task, index) => {
-              const isFocused = index === this.state.selectedIndex
-              const key = `${task.storageKey}-${task.isSelected}-${isFocused}`
-              return <TaskListItem {...task} key={key} isFocused={isFocused} />
-            })}
-          </ol>
+          {this.props.tasks.length > 0 ? (
+            <ol className="task-list">
+              {this.props.tasks.map((task, index) => {
+                const isFocused = index === this.state.selectedIndex
+                const key = `${task.storageKey}-${task.isSelected}-${isFocused}`
+                return (
+                  <TaskListItem
+                    {...task}
+                    key={key}
+                    isFocused={isFocused}
+                  />
+                )
+              })}
+            </ol>
+          ) : (
```

Comment:
I don't think it can, actually, not while staying mixed into the HTML here. I could make a function that does if/else in it and returns different chunks of HTML, then call the function here like `{this.listOrMessage()}`.